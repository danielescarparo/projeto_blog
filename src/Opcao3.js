import React, { Component } from 'react';

import './Opcao3.css';

class Opcao3 extends Component {
  render() {

    console.log(this.props);

    return (
      <div>
          Mudeid e novo
      </div>  
    );
  }
}

export default Opcao3;