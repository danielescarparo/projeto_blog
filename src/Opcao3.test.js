import React from 'react';
import ReactDOM from 'react-dom';
import Opcao3 from './Opcao3';
import Adapter from 'enzyme-adapter-react-16'; 
import { shallow, configure } from 'enzyme'; 
configure({adapter: new Adapter()});

it('renders without crashing', () => {
  const wrapper = shallow(<Opcao3/>);
  expect(wrapper.html()).toMatchSnapshot();
});

