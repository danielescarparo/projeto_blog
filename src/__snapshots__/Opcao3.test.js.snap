// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `"<div>Mudeid e novo</div>"`;
